pkgs/by-name/vl/vlc-bittorrent/package.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, autoconf-archive
+, autoreconfHook
+, boost
+, fetchFromGitHub
+, libtorrent-rasterbar
+, libvlc
+, openssl
+, pkg-config
+, stdenv
+}:
+
+# VLC does not know where the vlc-bittorrent package is installed.
+# make sure to have something like:
+#   environment.variables.VLC_PLUGIN_PATH = "${pkgs.vlc-bittorent}";
+
+stdenv.mkDerivation rec {
```

Suggestion: use `finalAttrs` to get rid of `rec`:

```suggestion
stdenv.mkDerivation (finalAttrs: {
```